dist/index.html

```diff
@@ -20,21 +19,20 @@ <h2>Recent Scores</h2>
           <!-- Scores Generated Dynamically with JS -->
         </ul>
```

Comment:
- [ ] You did a great job on the design, It really looks cool. But Now the name and the score are tight together without any space. Please use a colon and space for your score example `name: 500` as on the wireframe. It will improve the design and visibility of the users.